include/sound/sof.h

```diff
@@ -66,6 +66,8 @@ struct sof_dev_desc {
 	/* alternate list of machines using this configuration */
 	struct snd_soc_acpi_mach *alt_machines;
 
+	bool acpi_capable; /* platform capable to manage PM through ACPI */
```

Comment:
acpi_target_states_used ?

I feel acpi_capable is too broad.

Reply:
or ```use_acpi_target_states```?